api/cosoul/art/[artTokenId].ts

```diff
@@ -0,0 +1,62 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { adminClient } from '../../../api-lib/gql/adminClient';
+import { errorResponse, NotFoundError } from '../../../api-lib/HttpError';
+import { Awaited } from '../../../api-lib/ts4.5shim';
+
+const CACHE_SECONDS = 60 * 5;
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  try {
+    let artTokenId: number | undefined;
+    if (typeof req.query.artTokenId == 'string') {
+      artTokenId = parseInt(req.query.artTokenId);
+    }
+
+    assert(artTokenId, 'no token Id provided');
```

Comment:
the assert will throw a 500, so this should be removed with the throw below so we return a 404 instead.